### DISCUSSION TITLE
User's iPhone is not at home but RA says it is

### DISCUSSION THREAD

**Discussion by toreupfeet** [mKeRix/room-assistant#564] [Q&A]
RA is working rather well. I haven't tested it extensively, but I'd give it a thumbs up, surely.

I have a problem at the moment though. One of the iPhones it tracks has not been at the house for a few hours now, yet my master bedroom Pi Zero says it is still at the house.

```
global:
  instanceName: pi_zero_master_bedroom
  integrations:
    - homeAssistant
    - bluetoothClassic
homeAssistant:
  mqttUrl: 'mqtt://192.168.50.192:1883'
  mqttOptions:
    username: homeassistant
    password: SuperSecretPassword
bluetoothClassic:
  interval: 6
  scanTimeLimit: 5
  preserveState: false
  hciDeviceId:
    - 0
  addresses:
    - '00:00:00:00:00:00'
    - '00:00:00:00:00:00'    
    - '00:00:00:00:00:00'
```

Any insight as to the cause/remedy of this problem?

**Comment by mKeRix** [comment]
Interesting, it shouldn't do that! Does it show that in Home Assistant or did you check the room-assistant [API](https://www.room-assistant.io/guide/api.html)? Do you have multiple instances of room-assistant or just the one you posted the config from?

**Comment by toreupfeet** [comment]
It's the HA state that I'm referring to. I have used Developer Tools to change the states of the entities back to not_home/Away.
I'm running the most recent HA on a Pi4. I have the HA add-on and three Pi Zeros. I'll check the API the next time I am home.
